doc/development_process/release_process.rst

```diff
@@ -0,0 +1,321 @@
+
+Long-term support releases are designed to be supported for a longer than normal
+period and will be the basis for products and certification for various usages.
+
+An LTS release is made every 2 years and is branched and maintained
+independently from the mainline tree.
+
+An LTS release will be branched and maintained independently of the mainline
+tree.
+
+
+.. figure:: lts.png
+    :align: center
+    :alt: Long Term Support Release
+    :figclass: align-center
+    :width: 80%
+
+    Long Term Support Release
+
+Changes and fixes flow in both directions. However, changes from master to an
+LTS branch will be limited to fixes that apply to both branches and for existing
+features only.
+
+All fixes for an LTS branch that apply to the mainline tree are pushed to
+mainline as well.
+
+
+Auditable Code Base
+===================
+
+An auditable code base is to be established from a defined subset of Zephyr OS
+features and will be limited in scope. The LTS,  development tree, and the
+auditable code bases shall be kept in sync after the audit branch is created,
+but with a more rigorous process in place for adding new features into the audit
+branch used for certification.
+
+This process will be applied before new features move into the
+auditable code base.
+
+The initial and subsequent certification targets will be decided by the Zephyr project
+governing board.
+
+Processes to achieve selected certification will be determined by the Security and
+Safety Working Groups and coordinated with the TSC.
+
+
+Release Procedure
+******************
+
+This section documents the Release manager responsibilities so that it serves as
+a knowledge repository for Release managers.
+
+Milestones
+==========
+
+The following graphic shows the timeline of phases and milestones associated
+with each release:
+
+.. figure:: milestones.png
+    :align: center
+    :alt: Release Milestones
+    :figclass: align-center
+    :width: 80%
+
+    Release milestones
+
+This shows how the phases and milestones of one release overlap with those of
+the next release:
+
+
+.. figure:: milestones2.png
+    :align: center
+    :alt: Release Milestones
+    :figclass: align-center
+    :width: 80%
+
+    Release milestones with planning
+
+
+.. csv-table:: Milestone Description
+   :header: "Milestone", "Description", "Definition"
+   :widths: auto
+
+
+   "P0","Planning Kickoff","Start Entering Requirements"
+   "P1","","TSC Agrees on Major Features and Schedule"
+   "M0","Merge Window Open","All features, Sized, and AssignedMerge Window Is Opened"
+   "M1","M1 Checkpoint","Major Features Ready for Code Reviews |br| Test Plans Reviewed and Approved"
+   "M2","Feature Merge Window Close","Feature Freeze |br| Feature Development Complete (including Code Reviews and Unit Tests Passing) |br| P1 Stories Implemented |br| Feature Merge Window Is Closed |br| Test Development Complete |br| Technical Documentation Created/Updated and Ready for Review |br| CCB Control Starts"
```

Comment:
If you're still editing, you should break this row into multiple lines (as you did for the last row in this table)